int32 left_ticks    # Ticks seen on left encoder, incrementing or decrementing from startup. Positive is forward motion.
int32 right_ticks    # Ticks seen on right encoder, incrementing or decrementing from startup. Positive is forward motion.
